app/views/school_groups/advice/comparison_reports.html.erb

```diff
@@ -0,0 +1,152 @@
+                 storage_heaters: :change_in_storage_heaters_since_last_year
+               } %>
+            <% list.with_fuel_types t('school_groups.advice.comparison_reports.links.last_two_holidays'), reports: {
+                 electricity: :change_in_electricity_holiday_consumption_previous_holiday,
+                 gas: :change_in_gas_holiday_consumption_previous_holiday
+               } %>
+            <% list.with_fuel_types t('school_groups.advice.comparison_reports.links.same_holiday'), reports: {
+                 electricity: :change_in_electricity_holiday_consumption_previous_years_holiday,
+                 gas: :change_in_gas_holiday_consumption_previous_years_holiday
+               } %>
+            <% list.with_fuel_types t('school_groups.advice.comparison_reports.links.current_holiday'), reports: {
+                 electricity: :electricity_consumption_during_holiday,
+                 gas: :gas_consumption_during_holiday,
+                 storage_heaters: :storage_heater_consumption_during_holiday
+               } %>
+            <% list.with_link t('school_groups.advice.comparison_reports.links.upcoming_holiday'),
+                              report: :holiday_usage_last_year %>
+          <% end %>
+        <% end %>
+        <% grid.with_block cell_classes: 'pb-2', classes: %w[p-4 bg-light rounded] do %>
+          <h4><%= t('school_groups.advice.comparison_reports.lists.baseload') %></h4>
+          <%= render SchoolGroups::ComparisonReportListComponent.new(school_group: @school_group,
+                                                                     fuel_types: @fuel_types) do |list| %>
+            <% list.with_link t('school_groups.advice.comparison_reports.links.baseload_per_pupil'),
+                              report: :baseload_per_pupil,
+                              fuel_type: :electricity %>
+            <% list.with_link t('school_groups.advice.comparison_reports.links.recent_baseload'),
+                              report: :recent_change_in_baseload,
+                              fuel_type: :electricity %>
+            <% list.with_link t('school_groups.advice.comparison_reports.links.peak_use'),
+                              report: :electricity_peak_kw_per_pupil,
+                              fuel_type: :electricity %>
+            <% list.with_named t('school_groups.advice.comparison_reports.links.baseload_variation'),
+                               fuel_type: :electricity,
+                               reports: {
+                                 seasonal_baseload_variation:
+                                  t('school_groups.advice.comparison_reports.links.seasonal'),
+                                 weekday_baseload_variation:
+                                  t('school_groups.advice.comparison_reports.links.weekday')
+                               } %>
+          <% end %>
+        <% end if @fuel_types.include?(:electricity) %>
```

Comment:
Not seen this syntax before, had to look it up :-) Not sure if I like it or not, but better now I know what it does :-)

Reply:
It's just `x if y`. I've not used it that much in templates but Tristan suggested in a few places to reduce wrapping in `<% if... %><% end %>`. Maybe something to discuss around our preferences.